core/src/main/python/wlsdeploy/tool/discover/k8s_operator_filter.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ------------
+# Description:
+# ------------
+#  This is a WDT filter for primordial domain creation. It filters out all resources and
+#  apps deployments, leaving only the domainInfo and admin server in topology.
+#
+def filter_model(model):
+    __cleanup_topology(model)
+    __cleanup_resources(model)
+
+def __cleanup_resources(model):
+    if model and 'resources' in model:
+        resources = model['resources']
+
+    for delthis in [ 'PartitionWorkManager', 'Partition', 'ResourceGroup', 'ResourceGroupTemplate']:
```

Comment:
what about virtual hosts, resource manager and resource management?

Reply:
added